### PR TITLE
Reverting to 1.0 meta layers and removed verified boot

### PR DESCRIPTION
**Pull request by ygoyal18**
Reverting to 1.0 meta layers as we do not have support to maintain the mbl meta layers.

### CODE REVIEW

File: conf/local.conf.sample [hunk 1]
```diff
@@ -94,40 +98,3 @@ PATCHRESOLVE = "noop"
 RM_OLD_IMAGE = "1"
 INHERIT += "rm_work"
 CONF_VERSION = "1"
-
-export SSH_AUTH_SOCK
```
Comment (costanic) [inline conf/local.conf.sample:98]: some users may still want this line?

File: recipes-kernel/linux/files/extra-kernel-config.cfg [hunk 2]
```diff
@@ -1,3 +0,0 @@
-CONFIG_OVERLAY_FS=y
```
Comment (costanic) [inline recipes-kernel/linux/files/extra-kernel-config.cfg:1]: are these configs pulled in by some other means?  I think we still need these for normal wigwag upgrade scenarios, right?

File: conf/local.conf.sample [hunk 3]
```diff
@@ -61,9 +61,13 @@ ENABLE_UART = "1"
 
 #initramfs setup
 INITRAMFS_IMAGE = "ww-console-image-initramfs"
+INITRAMFS_IMAGE_BUNDLE = "1"
 BOOT_SPACE = "1073741"
 INITRAMFS_MAXSIZE = "2600000"
+#IMAGE_FSTYPES_pn-${INITRAMFS_IMAGE} = "${INITRAMFS_FSTYPES}"
```
Comment (cmonr) [inline conf/local.conf.sample:67]: Is this needed if it's commented out?

File: recipes-core/ww-console-image-initramfs-init/files/initramfs-init-script.sh [hunk 4]
```diff
@@ -570,7 +561,7 @@ echo "  \ V  V / | | (_| |\ V  V / (_| | (_| |"
 funccall(){
 	type $1 > /dev/null 2>&1
 	if [[ $? -eq 0 ]]; then
-		$1 "$@"
+		$1
```
Comment (cmonr) [inline recipes-core/ww-console-image-initramfs-init/files/initramfs-init-script.sh:564]: Won't omiting this line cause problems that including it fixed?

File: recipes-core/ww-console-image-initramfs-init/files/initramfs-init-script.sh [hunk 5]
```diff
@@ -664,34 +655,18 @@ determinebuildversions(){
...
-			_decho "No new $uboot_bin available in the boot upgrade data."
-			ls -la /tmpfs/*
-			tar tf /tmpfs/boot.tar.xz
-			_decho "Upgrade may fail if the ROT key has changed."
-		fi
-	fi
-
 	_decho "currentUboot='$currentUboot'"
 	_decho "availableUboot='$availableUboot'"
+	rm /uboot.img
```
Comment (cmonr) [inline recipes-core/ww-console-image-initramfs-init/files/initramfs-init-script.sh:669]: In this new version of the file uboot will now always be removed. Why is that now the correct behavior?

File: conf/local.conf.sample (hunk 1 above)
Comment (cmonr) [reply conf/local.conf.sample:98]: I think I agree with @costanic in that keeping this in would probably be fine.

**Comment by cmonr** [comment]
>  recipes-kernel/linux/linux-raspberrypi_%.bbappend 

This will also need to be rebased with the base branch before it can continue.

File: recipes-kernel/linux/files/extra-kernel-config.cfg (hunk 2 above)
Comment (ygoyal18) [reply recipes-kernel/linux/files/extra-kernel-config.cfg:1]: @x85446 how are we enabling overlayfs in 1.0?

File: recipes-kernel/linux/linux-raspberrypi_%.bbappend [hunk 6]
```diff
@@ -1,5 +1,4 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
 SRC_URI += "file://0001-Edge-increased-the-HCI_LE_AUTOCONN_TIMEOUT-to-20-sec.patch \
-            file://extra-kernel-config.cfg" \
```
Comment (cmonr) [inline recipes-kernel/linux/linux-raspberrypi_%.bbappend:4]: Was this no longer needed?

File: recipes-core/ww-console-image-initramfs-init/files/initramfs-init-script.sh (hunk 5 above)
Comment (ygoyal18) [reply recipes-core/ww-console-image-initramfs-init/files/initramfs-init-script.sh:669]: @cmonr  Even in the old version it was always being removed, see line 673.

**Comment by ygoyal18** [comment]
Sorry, merging this PR in order to release 2.0.0-RC1 over the weekend.

**Pull request merged**